GitForWindowsHelper/component-updates.js

```diff
@@ -109,27 +109,28 @@ const guessReleaseNotes = async (context, issue) => {
 
 const pacmanRepositoryBaseURL = 'https://wingit.blob.core.windows.net/'
 
-const pacmanRepositoryURLs = (package_name, version) =>
-    isMSYSPackage(package_name)
-        ? [
-            `${pacmanRepositoryBaseURL}i686/${package_name}-${version}-1-i686.pkg.tar.xz`,
-            `${pacmanRepositoryBaseURL}x86-64/${package_name}-${version}-1-x86_64.pkg.tar.xz`
-        ] : [
-            `${pacmanRepositoryBaseURL}i686/${package_name.replace(/^mingw-w64/, '$&-i686')}-${version}-1-any.pkg.tar.xz`,
-            `${pacmanRepositoryBaseURL}x86-64/${package_name.replace(/^mingw-w64/, '$&-x86_64')}-${version}-1-any.pkg.tar.xz`
-        ]
+const pacmanRepositoryURLs = (package_name, version, architectures) =>
+    architectures.map(arch => {
+        const fileName = isMSYSPackage(package_name)
+            ? `${package_name}-${version}-1-${arch}.pkg.tar.xz`
+            : `${package_name.replace(/^mingw-w64/, `$&-${arch}`)}-${version}-1-any.pkg.tar.xz`
+        return `${pacmanRepositoryBaseURL}${arch.replace(/_/g, '-')}/${fileName}`
+    })
 
 const getMissingDeployments = async (package_name, version) => {
     // MinTTY is at epoch 1, which is part of Pacman's versioning scheme
     if (package_name === 'mintty') version = `1~${version}`
+    const architectures = ['i686', 'x86_64']
+    if (package_name === 'msys2-runtime') architectures.shift()
+    else if (package_name === 'msys2-runtime-3.3') architectures.pop()
```

Comment:
We might need to revisit the `push()`/`shift()` approach when we add `aarch64` to the mix, but it should work for now.

Reply:
Yep, let's do that once the need arises. Probably something like `architectures.splice(0, Infinity, 'x86_64')` or turning the `const` to a `let` and reassigning.